plugins/qsi-dock/fu-qsi-dock-common.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2022 Kevin Chen <hsinfu.chen@qsitw.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+#define Report_ID 5
+
+#define TX_ISP_LENGTH 61
+#define TX_ISP_LENGTH_MCU 60
+#define EXTERN_FLASH_PAGE_SIZE 256
+
+#define FIRMWARE_IDX_NONE   0x00
+#define FIRMWARE_IDX_DMC_PD 0x01
+#define FIRMWARE_IDX_DP	    0x02
+#define FIRMWARE_IDX_TBT4   0x04
+#define FIRMWARE_IDX_USB3   0x08
+#define FIRMWARE_IDX_USB2   0x10
+#define FIRMWARE_IDX_AUDIO  0x20
+#define FIRMWARE_IDX_I225   0x40
+#define FIRMWARE_IDX_MCU    0x80
+
+typedef struct {
+	guint8 DMC[5];
+	guint8 PD[5];
+	guint8 DP5x[5];
+	guint8 DP6x[5];
+	guint8 TBT4[5];
+	guint8 USB3[5];
+	guint8 USB2[5];
+	guint8 AUDIO[5];
+	guint8 I255[5];
+	guint8 MCU[2];
+	guint8 bcdVersion[2];
+} IspVersionInMcu_t;
+
+enum Hid_CmdPrimary {
```

Comment:
this needs to be of the form:

    typedef enum {
        UPPERCASE_ENUM_NAMES = 0xAA,
    } FuQsiDockHidCmdPrimary;